src/components/grid-aware/DonationBlock/DonationBlock.js

```diff
@@ -22,50 +22,96 @@ const LGLForm = () => (
 
 const DonationBlock = ({
   mainTitle,
+  givingTuesdayImage,
+  givingTuesdayParagraph,
```

Comment:
Could you give these names that are less specific to Giving Tuesday and more general to the role which they play on this page?

Also, please pluralize the names of any props that hold an array of items. It's confusing that `givingTuesdayParagraph` is singular but contains an array of paragraphs.

Reply:
ok